src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public class ColorSupplier extends Lottery {
+    public static String getRandomColor() {
```

Comment:
Avoid using static methods as per the checklist. Instead, consider creating an instance of `ColorSupplier` and calling the method on that instance.